xigt/scripts/xigt_export.py

```diff
@@ -13,6 +13,7 @@ def run(infile, outpath, out_format, config=None):
         import xigt.exporters.latex as exporter
     elif out_format == 'itsdb':
         import xigt.exporters.itsdb as exporter
+        cfg = {'relations': relations}
```

Comment:
This will overwrite any config a user provided with the `--config` option. Better is something like this:

```python
        if not cfg:
            cfg = {}
        cfg['relations'] = relations
```